ui-v2/tests/unit/routes/application-test.js

```diff
@@ -2,7 +2,13 @@ import { moduleFor, test } from 'ember-qunit';
 
 moduleFor('route:application', 'Unit | Route | application', {
   // Specify the other units that are required for this test.
-  needs: ['service:repository/dc', 'service:settings'],
+  needs: [
+    'service:repository/dc',
+    'service:settings',
+    'service:feedback',
+    'service:flashMessages',
+    'service:logger',
+  ],
```

Comment:
You'll be happy to know you don't have to do this in the new style of testing 💃

Reply:
Oooh nice, I always felt it should just figure them all out itself, do you not need `needs` at all now then?

Reply:
Nope! The test setup is pretty different in general, but needs isn't ... needed anymore.